apps/account-api/test/delegations.controller.e2e-spec.ts

```diff
@@ -104,6 +105,52 @@ describe('Delegation Controller', () => {
       });
   });
 
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with invalid accountId', async () => {
+    const providerId = provider.msaId?.toString();
+    const { keypair } = users[1];
+    const invalidAccountId = `${keypair.address.slice(0, -1)}5`;
+    const getPath: string = `/v1/delegation/revokeDelegation/${invalidAccountId}/${providerId}`;
+    await request(httpServer).get(getPath).expect(400).expect({
+      statusCode: 400,
+      message: 'Invalid accountId',
+    });
+  });
+
+  it('(GET) /v1/delegation/revokeDelegation/:accountId/:providerId with valid accountId: no msa', async () => {
+    const providerId = provider.msaId?.toString();
+    const keyring = new Keyring({ type: 'sr25519' });
+    const keypair = keyring.addFromUri('//Bob');
```

Comment:
Perhaps best to not use `//Bob` here; changes to our chain scenario setup might provision a user for `//Bob` at some point. Since you don't need a token-holding user, you can use a derived URI like `//Bob//invalidKeypair`

Reply:
look to other tests where we get user[]

Reply:
All `user[]` have an msa. This test needs an `accountId` without an MSA.